bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -125,7 +139,7 @@ async def apply_infraction(
                 log.error(f"Failed to DM {user.id}: could not fetch user (status {e.status})")
             else:
                 # Accordingly display whether the user was successfully notified via DM.
-                if await _utils.notify_infraction(user, infr_type, expiry, reason, icon):
+                if await _utils.notify_infraction(user, infr_type, expiry, reason_override, icon):
```

Comment:
Should this be passed `reason` still and instead have `reason_override` assigned to `reason` if provided?

Reply:
Perhaps the name `reason_override` is a bit misleading, it's for having a different message be sent to the user as the `reason` for the infraction rather than the reason sent to `#mod-log`. I've renamed the parameter to hopefully make it clearer.

Reply:
Thanks, it's better now.